go/vt/vtgate/planbuilder/testdata/lock_cases.txt

```diff
@@ -16,7 +16,25 @@
     ]
   }
 }
-Gen4 plan same as above
+{
+  "QueryType": "SELECT",
+  "Original": "select get_lock('xyz', 10) from dual",
+  "Instructions": {
+    "OperatorType": "Lock",
+    "Keyspace": {
+      "Name": "main",
+      "Sharded": false
+    },
+    "TargetDestination": "KeyspaceID(00)",
+    "FieldQuery": "select get_lock('xyz', 10) from dual where 1 != 1",
+    "lock_func": [
+      "get_lock('xyz', 10)"
+    ]
+  },
+  "TablesUsed": [
+    "dual"
+  ]
+}
```

Comment:
other places it is <ks>.dual and in lock_cases it is only "dual", why this is different?

Reply:
found and fixed